tools/Dashboard/DevHome.Dashboard/Views/DashboardView.xaml.cs

```diff
@@ -151,6 +165,68 @@ private async Task InitializeDashboard()
         ViewModel.IsLoading = false;
     }
 
+    private async Task PinDefaultWidgets()
+    {
+        string[] defaultWidgetDefinitionIds =

```

Comment:
Not blocking but I think this list would be helpful to have as a constant defined outside of a method.

Reply:
Someday I'd still like to do a big refactor out of DashboardView and into DashboardViewModel, great thing to do during that change.